src/frontend/apps/desk/src/features/teams/team-management/components/InputTeamName.tsx

```diff
@@ -21,14 +23,53 @@ export const InputTeamName = ({
   label,
   setTeamName,
 }: InputTeamNameProps) => {
+  const { t } = useTranslation();
   const [isInputError, setIsInputError] = useState(isError);
 
+  const getCauses = (error: APIError): string[] => {
+    const handledCauses: string[] = [];
+    const unhandledCauses = parseAPIError({
+      error,
+      errorParams: {
+        slug: {
+          causes: ['Team with this Slug already exists.'],
+          handleError: () => {
+            handledCauses.push(
+              t(
+                'This name is already used for another group. Please enter another one.',
+              ),
+            );
+          },
+        },
+      },
+      serverErrorParams: {
+        defaultMessage: t(
+          'Your request cannot be processed because the server is experiencing an error. If the problem ' +
+            'persists, please contact our support to resolve the issue: suiteterritoriale@anct.gouv.fr',
+        ),
+      },
+    });
+
+    let causes: string[] = [];
+
+    if (handledCauses?.length) {
+      causes = [...handledCauses];
+    }
+    if (unhandledCauses?.length) {
+      causes = [...causes, ...unhandledCauses];
+    }
+
+    return causes;
+  };
```

Comment:
If find it is a bit complicated, what about that ?

```suggestion
  const dictCauses: { [key: string]: string } = {
    'Team with this Slug already exists.': t(
      'This name is already used for another group. Please enter another one.',
    ),
  };
  const causes = error?.cause?.map(
    (cause) =>
      dictCauses[cause] ||
      t(
        'Your request cannot be processed because the server is experiencing an error. If the problem ' +
          'persists, please contact our support to resolve the issue: suiteterritoriale@anct.gouv.fr',
      ),
  );
```